### PR TITLE
10.0 website mass mailing

### PR DESCRIPTION
**Pull request by gfcapalbo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website_mass_mailing/migrations/10.0.1.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    # these two fields have not changed model but have changed model
+    # they where previously defined in mass_mail now in website_mass_mail
+    # we restore them in their place on the same table
+    sql = "update mail_mass_mailing_list set popup_content = %s" % (
+        openupgrade.get_legacy_name('popup_content')
+    )
+    cr.execute(sql)
+    sql = "update mail_mass_mailing_list set popup_redirect_url = %s" % (
+        openupgrade.get_legacy_name('popup_redirect_url')
+    )
```
Comment (pedrobaeza) [inline addons/website_mass_mailing/migrations/10.0.1.0/post-migration.py:19]: This SQL is not executed.

Comment (gfcapalbo) [reply addons/website_mass_mailing/migrations/10.0.1.0/post-migration.py:19]: WIP!

**Comment by pedrobaeza** [comment]
Any news on this?

**Comment by pedrobaeza** [comment]
Hi, are you going to finish this?

**Comment by gfcapalbo** [comment]
Added the commit , checked the change log. Still testing but I think it is complete.

**Comment by pedrobaeza** [comment]
@gfcapalbo thanks for the update. Looking now at the migration script, I think there's actually nothing to do, as the columns are not dropped during OpenUpgrade mass_mailing update process, so they will be there when updating website_mass_mailing.

**Comment by StefanRijnhart** [comment]
Closing in favour of https://github.com/OCA/OpenUpgrade/pull/1526

**Pull request closed without merging**